### PR TITLE
Working remote image buffer

### PR DESCRIPTION
**Pull request by nikso-itu**

### CODE REVIEW

File: src/vmem/vmem_upload.c [hunk 1]
```diff
@@ -7,65 +7,118 @@
...
+	uint32_t _tail;
+	uint32_t _list[BUFFER_LIST_SIZE];
+	PARAM_DEFINE_REMOTE_DYNAMIC(PARAMID_BUFFER_LIST, buffer_list, node, PARAM_TYPE_UINT32, BUFFER_LIST_SIZE, sizeof(uint32_t), PM_REMOTE, &_list, NULL);
+	PARAM_DEFINE_REMOTE_DYNAMIC(PARAMID_BUFFER_HEAD, buffer_head, node, PARAM_TYPE_UINT32, -1, 0, PM_REMOTE, &_head, NULL);
+	PARAM_DEFINE_REMOTE_DYNAMIC(PARAMID_BUFFER_TAIL, buffer_tail, node, PARAM_TYPE_UINT32, -1, 0, PM_REMOTE, &_tail, NULL);
+    
+    /* Add remote parameters to local parameters */
+    param_list_add(&buffer_head);
+    param_list_add(&buffer_tail);
+    param_list_add(&buffer_list);
```
Comment (nikso-itu) [inline src/vmem/vmem_upload.c:55]: Remove

File: src/vmem/vmem_upload.c [hunk 2]
```diff
@@ -7,65 +7,118 @@
...
+}
 
+vmem_list2_t vmem_file = {0};
+uint8_t buffer_node_id = 0;
 void upload(unsigned char *data, int len)
 {
-    /* Fetch radio CSP node ID */
-    uint8_t radio_id = param_get_uint8(&radio_node_id);
+    uint8_t node = param_get_uint8(&radio_node_id);
+    printf("node: %d\n", node);
```
Comment (nikso-itu) [inline src/vmem/vmem_upload.c]: Remove

**Review by nikso-itu — COMMENTED** [review]
Remove prints

File: src/vmem/vmem_upload.c (hunk 2 above)
Comment (nikso-itu) [inline src/vmem/vmem_upload.c]: Remove print

**Review by KjellbergD — APPROVED** [review]

**Review by nikso-itu — COMMENTED** [review]
Good changes

**Review by Lindharden — APPROVED** [review]

**Pull request merged**